packages/axelar-wasm-std/src/address.rs

```diff
@@ -29,6 +31,13 @@ pub fn validate_address(address: &str, format: &AddressFormat) -> Result<(), Err
             SuiAddress::from_str(address)
                 .change_context(Error::InvalidAddress(address.to_string()))?;
         }
+        AddressFormat::Stellar => {
+            if address != address.to_uppercase() {
+                bail!(Error::InvalidAddress(address.to_string()))
+            }
+            ScAddress::from_str(address)
+                .change_context(Error::InvalidAddress(address.to_string()))?;
```

Comment:
can you measure the impact on bytecode increase and gas usage due to using stellar libraries? we should add some automation around this

Reply:
not much, around 20k bytes. don't think it will increase gas, we validate all source address. the conversion is not expensive.